### PR TITLE
Update mesosphere link

### PR DESCRIPTION
**Pull request by Moshe-Immerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +206,7 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [Mantl](https://github.com/mantl/mantl) - Mantl is a modern platform for rapidly deploying globally distributed services
 * [Marathon](https://mesosphere.github.io/marathon/docs/) - Marathon is a private PaaS built on Mesos. It automatically handles hardware or software failures and ensures that an app is "always on"
 * [MCollective Docker Agent](https://github.com/m4ce/mcollective-docker-agent) - Uses MCollective to orchestrate your Docker containers and images by  [@m4ce](https://github.com/m4ce)
+* [Mesosphere DC/OS](https://mesosphere.com/product/) - Integrated platform for data and containers by [@mesosphere](https://mesosphere.com)
```
Comment (veggiemonk) [inline README.md]: Replace with
```
[Apache Mesos](https://github.com/apache/mesos) - abstracts CPU, memory, storage, and other compute resources away from machines (physical or virtual), enabling fault-tolerant and elastic distributed systems to easily be built and run effectively. [Documentation](https://mesos.apache.org/documentation/latest/docker-containerizer/)
```

What do you think?

Comment (Moshe-Immerman) [reply README.md]: I think DC/OS does add enough on top of Mesos to deserve inclusion, how about this: 
```md
[Mesosphere DC/OS](https://mesosphere.com/product/) - Integrated platform for data and containers built on top of [Apache Mesos](https://mesos.apache.org) by [@mesosphere](https://mesosphere.com)```

Comment (Moshe-Immerman) [reply README.md]: Or Split into 2 lines: 
- Apache Mesos
- DC/OS

Comment (veggiemonk) [reply README.md]: Add the $$$ or 💲or 💵 or 💰 or 🤑
Or maybe it should be moved to the service section?

**Pull request merged**